test/unit/test_blas_tile/test_gemm.h

```diff
@@ -93,14 +84,9 @@ void testGemm(blas::Op op_a, blas::Op op_b, SizeType m, SizeType n, SizeType k,
     return beta * el_c(index) + gamma * TypeUtilities<T>::polar((i + 1) / (j + 2), 2 * i + j);
   };
 
-  // Set tile elements.
-  set(a0, el_op_a, op_a);
-  set(b0, el_op_b, op_b);
-  set(c, el_c);
-
-  // Read-only tiles become constant if CT is const T.
-  Tile<CT, Device::CPU> a(std::move(a0));
-  Tile<CT, Device::CPU> b(std::move(b0));
+  Tile<CT, Device::CPU> a = createTile<CT>(el_op_a, size_a, lda, op_a);
```

Comment:
I'm wondering if 
```suggestion
  auto a = createTile<CT>(el_op_a, size_a, lda, op_a);
```
would be better.
What do you think?

Reply:
It's more compact and helps to avoid mistakes while writing the whole type. I will substitute them all, if you agree.